03-transactions.md

```diff
@@ -698,73 +698,408 @@ HTLC-success) gives a weight of:
+    # local_signature = 30440220764db825c7bebe8834cb6fdb811385b674d44c280b771d3bd116a86348a1fb4a02201b4eae2b72a64a21a6f4e07ac4e3c75ae52cc2d5e791d02c1174c0333dc89f2c
+    output htlc_success_tx 1: 020000000001015c677ff0b1ba939728ae4c2fa99f88f9756620c0040a6c46bc4faa2d4e48260f01000000000000000001d0070000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500483045022100944aeff8b05350a01b7a53fe70c47d7f7c65781cf8162623be9fe2652c809ff20220444b7945226b84fe180fc86f078f44fa5c9a23e481d20b748ff17d8f4c67cdb1014730440220764db825c7bebe8834cb6fdb811385b674d44c280b771d3bd116a86348a1fb4a02201b4eae2b72a64a21a6f4e07ac4e3c75ae52cc2d5e791d02c1174c0333dc89f2c012001010101010101010101010101010101010101010101010101010101010101016e21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c8201208763a9144b6b2e5444c2639cc0fb7bcea5afba3f3cdce23988527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae6775029000b175ac6800000000
+    # local_signature = 3045022100f7bbe202fa6ee83cc745e13e0d18eaf7f738a604122ce91e8a370cf53748ed130220011f0001b6b2cdc44041278fe4f3cfe27aeea2742ac7ae7b47f1dad9b77f13a8
+    output htlc_timeout_tx 2: 020000000001015c677ff0b1ba939728ae4c2fa99f88f9756620c0040a6c46bc4faa2d4e48260f02000000000000000001d0070000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e05004830450221009bb78d76c1b3bab0e189ad8d00497b422ccb68279d16b41b59c87e7d88351c630220636f8f9530cc116f23e65c1b3bbaa85183a139ae1ad598e1948e3614c76bd43401483045022100f7bbe202fa6ee83cc745e13e0d18eaf7f738a604122ce91e8a370cf53748ed130220011f0001b6b2cdc44041278fe4f3cfe27aeea2742ac7ae7b47f1dad9b77f13a801006e21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c8201208763a914b43e1b38138a41b37f7cd9a1d274bc63e3a9b5d188527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae6775029000b175ac6822a10700
+    # local_signature = 3044022066b026dae28aae6b9beb3c4ce1ea5433ed47e91331cc44a2294baa8d733319c702201df6a6284d1eb6f47f58e7ccd9e1c03feef32ac30a56085da1aa5ac673ca36dd
+    output htlc_timeout_tx 3: 020000000001015c677ff0b1ba939728ae4c2fa99f88f9756620c0040a6c46bc4faa2d4e48260f03000000000000000001b80b0000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500483045022100ab3b9a12b19f4dce462fa595c941665e0c94ca09c71d82c8dcfba82d2e11e32002201b644aa85fda740c49876e2ba032a50492672170feb0165969e739b75911730501473044022066b026dae28aae6b9beb3c4ce1ea5433ed47e91331cc44a2294baa8d733319c702201df6a6284d1eb6f47f58e7ccd9e1c03feef32ac30a56085da1aa5ac673ca36dd01006e21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c8201208763a9148a486ff2e31d6158bf39e2608864d63fefd09d5b88527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae6775029000b175ac6823a10700
```

Comment:
htlc timeout txs seem to have a witness stack that ends with a "received HTLC script", shouldn't it be an "offered HTLC" script